[2022-05-17 20:23:11,465] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: httpChartDag.inaccurate scheduled__2022-05-16T00:00:00+00:00 [queued]>
[2022-05-17 20:23:11,475] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: httpChartDag.inaccurate scheduled__2022-05-16T00:00:00+00:00 [queued]>
[2022-05-17 20:23:11,475] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-05-17 20:23:11,475] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-05-17 20:23:11,476] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-05-17 20:23:11,488] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): inaccurate> on 2022-05-16 00:00:00+00:00
[2022-05-17 20:23:11,491] {standard_task_runner.py:52} INFO - Started process 6025 to run task
[2022-05-17 20:23:11,494] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'httpChartDag', 'inaccurate', 'scheduled__2022-05-16T00:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/httpChartDag.py', '--cfg-path', '/tmp/tmphc0h34q5', '--error-file', '/tmp/tmp99_l6bef']
[2022-05-17 20:23:11,495] {standard_task_runner.py:80} INFO - Job 13: Subtask inaccurate
[2022-05-17 20:23:11,540] {task_command.py:369} INFO - Running <TaskInstance: httpChartDag.inaccurate scheduled__2022-05-16T00:00:00+00:00 [running]> on host 9b3fb5fadbaa
[2022-05-17 20:23:11,600] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=httpChartDag
AIRFLOW_CTX_TASK_ID=inaccurate
AIRFLOW_CTX_EXECUTION_DATE=2022-05-16T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-16T00:00:00+00:00
[2022-05-17 20:23:11,601] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-17 20:23:11,602] {subprocess.py:74} INFO - Running command: ['bash', '-c', "echo 'inaccurate'"]
[2022-05-17 20:23:11,609] {subprocess.py:85} INFO - Output:
[2022-05-17 20:23:11,614] {subprocess.py:92} INFO - inaccurate
[2022-05-17 20:23:11,615] {subprocess.py:96} INFO - Command exited with return code 0
[2022-05-17 20:23:11,638] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=httpChartDag, task_id=inaccurate, execution_date=20220516T000000, start_date=20220517T202311, end_date=20220517T202311
[2022-05-17 20:23:11,667] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-17 20:23:11,691] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
